src/core/hle/service/am/applets/software_keyboard.cpp

```diff
@@ -1,200 +1,1076 @@
+}
+
+void SoftwareKeyboard::RequestCalc(const std::vector<u8>& request_data) {
+    LOG_DEBUG(Service_AM, "Processing Request: Calc");
+
+    ASSERT(request_data.size() == sizeof(SwkbdRequestCommand) + sizeof(SwkbdCalcArg));
+
+    std::memcpy(&swkbd_calc_arg, request_data.data() + sizeof(SwkbdRequestCommand),
+                sizeof(SwkbdCalcArg));
+
+    if (swkbd_calc_arg.flags.set_input_text) {
+        current_text = Common::UTF16StringFromFixedZeroTerminatedBuffer(
+            swkbd_calc_arg.input_text.data(), swkbd_calc_arg.input_text.size());
+    }
+
+    if (swkbd_calc_arg.flags.set_cursor_position) {
+        current_cursor_position = swkbd_calc_arg.cursor_position;
+    }
+
+    if (swkbd_calc_arg.flags.set_utf8_mode) {
+        inline_use_utf8 = swkbd_calc_arg.utf8_mode;
+    }
+
+    if (swkbd_state <= SwkbdState::InitializedIsHidden &&
+        swkbd_calc_arg.flags.unset_customize_dic) {
+        ReplyUnsetCustomizeDic();
     }
+
+    if (swkbd_state <= SwkbdState::InitializedIsHidden &&
+        swkbd_calc_arg.flags.unset_user_word_info) {
+        ReplyReleasedUserWordInfo();
+    }
+
+    if (swkbd_state == SwkbdState::NotInitialized && swkbd_calc_arg.flags.set_initialize_arg) {
+        InitializeFrontendKeyboard();
+
+        ChangeState(SwkbdState::InitializedIsHidden);
+
+        ReplyFinishedInitialize();
+    }
+
+    if (!swkbd_calc_arg.flags.set_initialize_arg &&
+        (swkbd_calc_arg.flags.set_input_text || swkbd_calc_arg.flags.set_cursor_position)) {
+        InlineTextChanged();
+    }
+
+    if (swkbd_state == SwkbdState::InitializedIsHidden && swkbd_calc_arg.flags.appear) {
+        ShowInlineKeyboard();
+        return;
+    }
+
+    if (swkbd_state == SwkbdState::InitializedIsShown && swkbd_calc_arg.flags.disappear) {
+        HideInlineKeyboard();
+        return;
+    }
+}
+
+void SoftwareKeyboard::RequestSetCustomizedDictionaries(const std::vector<u8>& request_data) {
+    LOG_WARNING(Service_AM, "SetCustomizedDictionaries is not implemented.");
+}
+
+void SoftwareKeyboard::RequestUnsetCustomizedDictionaries(const std::vector<u8>& request_data) {
+    LOG_WARNING(Service_AM, "(STUBBED) Processing Request: UnsetCustomizedDictionaries");
+
+    ReplyUnsetCustomizedDictionaries();
+}
+
+void SoftwareKeyboard::RequestSetChangedStringV2Flag(const std::vector<u8>& request_data) {
+    LOG_DEBUG(Service_AM, "Processing Request: SetChangedStringV2Flag");
+
+    ASSERT(request_data.size() == sizeof(SwkbdRequestCommand) + 1);
+
+    std::memcpy(&use_changed_string_v2, request_data.data() + sizeof(SwkbdRequestCommand), 1);
+}
+
+void SoftwareKeyboard::RequestSetMovedCursorV2Flag(const std::vector<u8>& request_data) {
+    LOG_DEBUG(Service_AM, "Processing Request: SetMovedCursorV2Flag");
+
+    ASSERT(request_data.size() == sizeof(SwkbdRequestCommand) + 1);
+
+    std::memcpy(&use_moved_cursor_v2, request_data.data() + sizeof(SwkbdRequestCommand), 1);
+}
+
+/// Inline Software Keyboard Replies
```

Comment:
```suggestion
// Inline Software Keyboard Replies
```

Otherwise this is a doxygen comment